var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -173,7 +173,7 @@ class Trilinos(CMakePackage):
     # deprecated SuperLUDist v3.3 together with an Amesos patch is working.
     conflicts('+superlu-dist', when='@11.4.1:11.14.3')
     # PnetCDF was only added after v12.10.1
-    conflicts('+pnetcdf', when='@:12.10.1')
+    conflicts('+pnetcdf', when='@11.4.1:12.10.1')
```

Comment:
Every time someone adds an older version of `trilinos`, this number will need to be changed. If you're only trying to exclude `master`, why not set the lower bound to 0?
```
$ spack python
Spack version 0.10.0
Python 2.7.13, Darwin x86_64
>>> from spack.version import *
>>> Version('master') < Version('0')
True
```